tests/console/pam.pm

```diff
@@ -29,7 +29,7 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
-    die "This test module is not enabled for openSUSE Leap yet" if is_leap();
+    die "This test module is not enabled for openSUSE Leap yet" if is_leap('<15.4');
```

Comment:
I actually wonder, what happens with leap 15.3. I don't think it's gonna be the end of the world. (although past me, seems to not have said anything about that... looking at the review of #8839

Reply:
15.3 works as well:
http://kazhua.qa.suse.de/tests/97